docs/index.rst

```diff
@@ -39,14 +39,8 @@ out our :doc:`Getting Started <startingguide>` guide. Development is
 supported by the National Science Foundation through grants AGS-1344155,
 OAC-1740315, AGS-1901712.
 
-MetPy follows `semantic versioning <https://semver.org>`_ in its version number. With our
-current 0.x version, that implies that MetPy's APIs (application programming interfaces) are
```

Comment:
Probably want to roll most of this back, with the exception of the updated gallery link.